src/pubsub/node_hometree.erl

```diff
@@ -82,3 +80,26 @@ node_to_path(Node) ->
 
 path_to_node([]) -> <<>>;
 path_to_node(Path) -> mongoose_bin:join([<<"">> | Path], <<"/">>).
+
+continue_init(Host, ServerHost, Owner) ->
```

Comment:
Did this work before the change? Or was it broken all the time?

Reply:
I did not confirm it 100%, because I was changing it together with some other parts at the time, but yeah, I think it was broken. Or worked by luck by using some defaults (in the tests for example, I don't know how well, but it _is_ tested...).

I tried to check it by matching `mod_pubsub:create_node` in the older version of this module to `{result, _}`, and I think it logged failures then.